File: svc-event/src/main/java/com/beancounter/event/CloudConfig.java
package com.beancounter.event;

import javax.annotation.PostConstruct;
import lombok.extern.slf4j.Slf4j;
import org.springframework.cloud.openfeign.EnableFeignClients;
import org.springframework.context.annotation.Configuration;

/**
 * Spring Cloud.
 * @author mikeh
 * @since 2019-03-03
 */
@Configuration
@EnableFeignClients
@Slf4j
public class CloudConfig {
  @PostConstruct
  void loaded() {
    log.info("Loaded");
  }
}

File: svc-event/src/main/kotlin/com/beancounter/event/CloudConfig.kt
package com.beancounter.event

import jakarta.annotation.PostConstruct
import org.slf4j.LoggerFactory
import org.springframework.context.annotation.Configuration

/**
 * Spring Cloud.
 *
 * @author mikeh
 * @since 2019-03-03
 */
@Configuration
class CloudConfig {
    @PostConstruct
    fun loaded() {
        log.info("Loaded")
    }

    companion object {
        private val log = LoggerFactory.getLogger(CloudConfig::class.java)
    }
}